specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/LinkedService.json

```diff
@@ -532,23 +558,23 @@
             "Online",
             "OnPremisesWithIfd"
           ],
-          "type": "string",
```

Comment:
What's the reason of this change?

Reply:
Agree. For enumerations, we should use string.